epictrack-api/src/api/resources/indigenous_nation.py

```diff
@@ -110,10 +111,10 @@ def get():
         indigenous_nations = IndigenousNationService.find_all_indigenous_nations(
             args.get("is_active")
         )
+        response = res.IndigenousResponseNationSchema(many=True).dump(indigenous_nations)
+        response = natural_sort(response, "name")
```

Comment:
Shall we do this inside the service?

Reply:
The `natsort` library does not support sorting of objects. We either need to do `res.IndigenousResponseNationSchema(many=True).dump(indigenous_nations)` inside the service, or keep `natural_sort` call in the resource.